CMake/portability/BlueGenePortability.cmake

```diff
@@ -1,66 +0,0 @@
-# Copyright (c) 2016, Blue Brain Project
```

Comment:
my heart is broken about the removing of this file :)

Reply:
You can't beat BlueGene ;)